src/main/kotlin/com/dcd/server/infrastructure/global/thirdparty/docker/DockerCommandExecutor.kt

```
@@ -214,5 +223,107 @@ class DockerCommandExecutor(
+                // 임시 컨테이너 시작 및 명령 실행
+                dockerClient.startContainerCmd(tempContainerName).exec()
+
+                // 명령 실행 완료 대기
+                val statusCode = dockerClient.waitContainerCmd(tempContainerName)
+                    .exec(WaitContainerResultCallback())
+                    .awaitStatusCode()
+                if (statusCode != 0) {
+                    deleteVolume(targetVolume)
+                    throw VolumeCopyFailureException()
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)